openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -99,15 +95,74 @@ def process(self, instance):
             "files": os.path.basename(mov_path),
             "stagingDir": staging_dir,
             "frameStart": 1,
-            "frameEnd": 1,
+            "frameEnd": img_number,
             "fps": 25,
             "preview": True,
             "tags": self.mov_options['tags']
         })
 
         # Required for extract_review plugin (L222 onwards).
         instance.data["frameStart"] = 1
-        instance.data["frameEnd"] = 1
+        instance.data["frameEnd"] = img_number
         instance.data["fps"] = 25
 
         self.log.info(f"Extracted {instance} to {staging_dir}")
+
+    def _get_image_path_from_instances(self, instance):
+        img_list = []
+
+        for instance in instance.context:
+            if instance.data["family"] != "image":
+                continue
+
+            for rep in instance.data["representations"]:
```

Comment:
```suggestion
            for rep in instance.data["representations"]:
                if "jpg" not in rep["files"]:
                    continue
```
I think it should be like this, because if you are producing both `png` and `jpg`, which is default, resulting mov has more frames.
![Screenshot 2022-02-07 214555](https://user-images.githubusercontent.com/4457962/152871808-153ea86f-a314-4a5b-bec3-75e4fc1bd166.png)

Reply:
Absolutely, in my configuration I only produce jpg! If a person does not produce the jpgs, the suggestion will not work!

Reply:
True, so there should be some more intelligent deduplication.